File: src/main/java/com/ocds/tender/model/dto/tender/Lot.java
package com.ocds.tender.model.dto.tender;

import lombok.Data;

import java.util.List;

@Data
public class Lot {
    public String id;
    public String title;
    public String description;
    public String status;
    public Value value;
    public List<Options> options;
    public RecurrentProcurement recurrentProcurement;
    public List<Renewals> renewals;
    public List<Variants> variants;
}

File: src/main/kotlin/com/procurement/access/model/dto/ocds/Lot.kt
package com.procurement.access.model.dto.ocds

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Lot @JsonCreator constructor(

        val id: String,

        var title: String?,

        var description: String?,

        var status: LotStatus?,

        var statusDetails: LotStatusDetails?,

        val value: Value,

        val options: List<Option>?,

        val recurrentProcurement: List<RecurrentProcurement>?,

        val renewals: List<Renewal>?,

        val variants: List<Variant>?,

        var contractPeriod: ContractPeriod?,

        var placeOfPerformance: PlaceOfPerformance?
)